examples/checkout/search/search_abandoned_transactions.rb

```diff
@@ -8,7 +8,10 @@
 #   You can pass the credentials parameters to PagSeguro::Transaction#find_abandoned
 
 options = {
-  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN')
+  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN'),
+  starts_at: (Time.now - 86400),
```

Comment:
what is 86400, 900 ? 
it may be better add these values in variables with names representing the values